### PR TITLE
feat(FX-3130): adjust styles on fair organizer page

### PR DESCRIPTION
**Pull request by anastasiapyzhik**
[FX-3130]

While QA'ing [this PR](https://github.com/artsy/force/pull/8096) @dblandin noticed some inconsistency with the design spec:

- [x] Increase line height of rendered about text
- [x] Reposition organizer name/date to right of logo
- [x] Update spacing/typography of “opens in” text 
- [x] Top-align about and “opens in” text. I assume this would top-align with the follow button if there isn’t a future show available.

Current PR fixes inconsistencies mentioned above

<details>
<summary>Before</summary>

Screenshot 2021-08-11 at 13 49 51
</details>

<details>
<summary>After</summary>

Screenshot 2021-08-11 at 13 51 12
</details>

[FX-3130]: https://artsyproduct.atlassian.net/browse/FX-3130

### CODE REVIEW

File: src/v2/Apps/FairOrginizer/Components/FairOrganizerHeader/FairOrganizerHeader.tsx [hunk 1]
```diff
@@ -14,31 +16,41 @@ interface FairOrganizerHeaderProps {
...
+          <Text as="h1" variant="xl">
+            Explore {name} on Artsy
+          </Text>
+          <Text variant="xl" color="black60" mb={1}>
+            {exhibitionPeriod}
+          </Text>
+        </Box>
+      </Flex>
+
+      <Spacer mt={75} />
```
[the remark this answers is not in the document]
Comment (anastasiapyzhik) [reply src/v2/Apps/FairOrginizer/Components/FairOrganizerHeader/FairOrganizerHeader.tsx:40]: @dblandin there are 2 spacers by 40px in the spec, maybe that'd be better to use 2 <Spacer mt={4} or there should only be one spacer component with the token that matches the most?

Screenshot 2021-08-11 at 16 19 32

[the remark this answers is not in the document]
Comment (dblandin) [reply src/v2/Apps/FairOrginizer/Components/FairOrganizerHeader/FairOrganizerHeader.tsx:40]: Hmmm, not sure. Let's merge for now and we can double-check with Jonathan on the right value here.

**Comment by dzucconi** [comment]
The auction timer is a shared component so all of these changes have to be validated at every place the component is used. This visually breaks the auction timer on the artwork page.

**Comment by dzucconi** [comment]
Filed a ticket: https://artsyproduct.atlassian.net/browse/FX-3189

File: src/v2/Apps/FairOrginizer/Components/FairOrganizerHeader/FairOrganizerHeader.tsx [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import React from "react"
+import moment from "moment"
```
Comment (dzucconi) [inline src/v2/Apps/FairOrginizer/Components/FairOrganizerHeader/FairOrganizerHeader.tsx:2]: Lump moment in with underscore as forbidden dependencies in new code and please call out their usage if you see this in code review. You can replace this with luxon.

**Pull request merged**